packages/flutter/lib/src/material/chip.dart

```diff
@@ -388,8 +391,8 @@ abstract class TappableChipAttributes {
 /// Supplying a non-null [onDeleted] callback will cause the chip to include a
 /// button for deleting the chip.
 ///
-/// Requires one of its ancestors to be a [Material] widget. The [label],
-/// [deleteIcon], and [border] arguments must not be null.
+/// Requires one of its ancestors to be a [Material] widget. The [label]
+/// argument must not be null.
```

Comment:
label and clipBehavior arguments must not be null

Reply:
Done